Sources/TuistKit/Lint/CodeLinter.swift

```diff
@@ -0,0 +1,57 @@
+import Foundation
+import TSCBasic
+import TuistCore
+import TuistSupport
+
+protocol CodeLinting {
+    func lint(sources: AbsolutePath, path: AbsolutePath) throws
+}
+
+class CodeLinter: CodeLinting {
```

Comment:
This class is very unit-testable so I'd add some tests:
Given the given `sources` and `path`, I expect `CodeLinter` to run the following system command: `xxx`.